src/modules/todoList.js

```diff
@@ -0,0 +1,46 @@
+export default class TodoList {
+    todos = [];
+
+     addTodo = (description) => {
+       const todo = {
+         description,
+         index: this.todos.length + 1,
+         completed: false,
+         id: Date.now(),
+       };
+       this.todos.push(todo);
+       this.save();
+     };
```

Comment:
- [Optional] In javascript it is good practice to use method chaining. return `this` at the end of every function so that it will increase the usability further of your code. And you can chain further class methods onto it.
- You can consider going through this [article](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) 
for further details about javascript best practices.
-
![Screenshot from 2022-02-22 00-29-15](https://user-images.githubusercontent.com/60083437/155010039-9d53ea0f-8a04-4a44-924e-3da3f946367b.png)

Reply:
The article was really helpful @Tufahel thank you.